dashboard/lib/firebase_options.dart

```diff
@@ -0,0 +1,76 @@
+// File generated by FlutterFire CLI.
+// ignore_for_file: lines_longer_than_80_chars, avoid_classes_with_only_static_members
+import 'package:firebase_core/firebase_core.dart' show FirebaseOptions;
+import 'package:flutter/foundation.dart' show defaultTargetPlatform, kIsWeb, TargetPlatform;
+
+/// Default [FirebaseOptions] for use with your Firebase apps.
+///
+/// Example:
+/// ```dart
+/// import 'firebase_options.dart';
+/// // ...
+/// await Firebase.initializeApp(
+///   options: DefaultFirebaseOptions.currentPlatform,
+/// );
+/// ```
+class DefaultFirebaseOptions {
+  static FirebaseOptions get currentPlatform {
+    if (kIsWeb) {
+      return web;
+    }
+    switch (defaultTargetPlatform) {
+      case TargetPlatform.android:
+        return android;
+      case TargetPlatform.iOS:
+        return ios;
+      case TargetPlatform.macOS:
+        return macos;
+      case TargetPlatform.windows:
+        throw UnsupportedError(
+          'DefaultFirebaseOptions have not been configured for windows - '
+          'you can reconfigure this by running the FlutterFire CLI again.',
+        );
+      case TargetPlatform.linux:
+        throw UnsupportedError(
+          'DefaultFirebaseOptions have not been configured for linux - '
+          'you can reconfigure this by running the FlutterFire CLI again.',
+        );
+      default:
+        throw UnsupportedError(
+          'DefaultFirebaseOptions are not supported for this platform.',
+        );
+    }
+  }
+
+  static const FirebaseOptions web = FirebaseOptions(
+    apiKey: 'AIzaSyCenA4U4l5CNiLJZDca2zlS0TrX8H7bV9s',
+    appId: '1:790958376406:web:abd3f6e0f2d7e77a439077',
+    messagingSenderId: '790958376406',
+    projectId: 'google.com:flutter-roll-on-borg',
```

Comment:
One issue is the flutter-dashboard project is locked down, and so I'm not sure what's needed. Since this is only for debugging purposes on crashes, I think we should save the time.

If long term we want to move Cocoon to firebase, revisiting this SGTM.

Reply:
Please hold a bit and I will give it a try. This seems an easy config change, and `flutter-dashboard` still makes more sense to me. Otherwise, we can go with the release project.

Reply:
https://firebase.corp.google.com/project/flutter-dashboard/overview is ready.